.travis.yml

```
@@ -0,0 +1,53 @@
+### Project specific config ###
+
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      env: ATOM_CHANNEL=stable
+
+    - os: linux
+      env: ATOM_CHANNEL=beta
+
+env:
+  global:
+    APM_TEST_PACKAGES: "language-lua"
+
+before_install:
```

```suggestion
install:
```
The `install` step runs before the `script` step, no reason not to just use it here.